src/docker.rs

```diff
@@ -19,6 +19,10 @@ pub fn build_container(docker_env: &str) -> Result<()> {
         .run()
 }
 
+pub fn is_running() -> bool {
```

Comment:
Can you mark this as `pub(crate)` instead of `pub`? I started to migrate the codebase to the former because with `pub` rustc doesn't always emit unused code warnings.